### PR TITLE
Add release drafter, cs fix and auto deployment of the archive via travi

### PR DESCRIPTION
**Pull request by jolelievre**
Add release drafter, cs fix and auto deployment of the archive via travi

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
   "require": {
     "php": ">=5.4"
   },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^2.15.1",
+    "prestashop/php-coding-standards": "dev-master"
```
Comment (Quetzacoalt91) [inline composer.json]: Lib is now 

```suggestion
    "prestashop/php-dev-tools": "^2"
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+language: php
+
+php:
+  - 5.6
+  - 7.2
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
```
Comment (Quetzacoalt91) [inline .travis.yml:10]:
```suggestion

install:
  - composer install
  
```

**Comment by jolelievre** [comment]
@Quetzacoalt91 I addressed your modifications, added a few missing files, and the build is now green
Can we merge it in dev without the API key? It is only used for a merge on master right?

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Yes the API key is needed only when creating tags, this can wait.

**Comment by Quetzacoalt91** [comment]
Thank you @jolelievre

**Pull request merged**